ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -306,23 +307,26 @@ def update_sample_duplication_and_coverage(project_id, sample_id, project_base_p
     """
     
     dup_file_path=os.path.join(project_base_path, 'ANALYSIS', project_id, 'piper_ngi', '05_processed_alignments', "{}.metrics".format(sample_id))
-    cov_file_path=os.path.join(project_base_path, 'ANALYSIS', project_id, 'piper_ngi', '06_final_alignment_qc', "{}.clean.dedup.qc".format(sample_id),"genome_results.txt")
+    genome_results_file_path=os.path.join(project_base_path, 'ANALYSIS', project_id, 'piper_ngi', '06_final_alignment_qc', "{}.clean.dedup.qc".format(sample_id),"genome_results.txt")
 
     try:
         dup_pc=parse_deduplication_percentage(dup_file_path)
     except:
         dup_pc=0
         LOG.error("Cannot find {}.metrics file for duplication rate at {}. Continuing.".format(sample_id, dup_file_path))
     try:
-        cov=parse_qualimap_coverage(cov_file_path)
-    except:
+        cov=parse_qualimap_coverage(genome_results_file_path)
+        reads=parse_qualimap_reads(genome_results_file_path)
+    except Exception as e:
```

Comment:
IOError, then I guess ? the previous version caught everything, but I don't think there's many other possible exceptions.